src/accelerate/utils/dataclasses.py

```diff
@@ -215,7 +213,12 @@ class DeepSpeedPlugin:
     This plugin is used to integrate DeepSpeed.
     """
 
-    config_file: str = field(default=None, metadata={"help": "Path to the DeepSpeed config file."})
+    hf_ds_config: Any = field(
```

Comment:
This is breaking. While the API was marked as experimental and we can do such a change, I'd like to understand why we can't keep the old argument name here.

Reply:
Hello, this isn't breaking change as this feature was added in PR from couple days ago #405 . It isn't part of the latest release we have and as such won't be a breaking change. The renaming was done to clearly explain that it isn't just limited to file and can accept either file, dict or `HFDeepSpeedConfig` Object.

Reply:
Oh okay then, I thought it had been there for longer.